cookbooks/bcpc/templates/default/apache-graphite-web.conf.erb

```diff
@@ -27,7 +28,7 @@ NameVirtualHost <%="#{node[:bcpc][:management][:ip]}"%>:443
         Allow from all
     </Directory>
 
-    Alias /content/ /opt/graphite/webapp/content/
+    Alias /static/ /opt/graphite/webapp/content/
```

Comment:
The graphite webapp wants to use /static for all its static media needs, it may have been content at some point 
compare https://github.com/graphite-project/graphite-web/blob/0.9.10/examples/example-graphite-vhost.conf to https://github.com/graphite-project/graphite-web/blob/master/examples/example-graphite-vhost.conf

However content is still th eon disk directory where media is being consolidated via django-admin collectstatic